assets/less/zenodo-rdm/elements/segment.overrides

```diff
@@ -28,16 +28,24 @@
       padding: 0;
     }
   }
+
+  .ui.placeholder {
+    &,
```

Comment:
& will resolve to the parent, so you could use the parent directly. Unless it is a syntax sugar :)

Reply:
The thing is that I want it also to apply to the parent, without defining it twice (on the parent, and on the children selectors)